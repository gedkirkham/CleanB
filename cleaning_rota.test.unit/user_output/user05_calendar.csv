Date,
25/08/2018,
01/09/2018,
08/09/2018,
15/09/2018,
22/09/2018,
29/09/2018,
06/10/2018,
13/10/2018,
20/10/2018,
27/10/2018,
03/11/2018,
10/11/2018,
17/11/2018,
24/11/2018,
01/12/2018,
08/12/2018,
15/12/2018,
22/12/2018,
25/08/2018,
01/09/2018,
08/09/2018,
15/09/2018,
22/09/2018,
29/09/2018,
06/10/2018,
13/10/2018,
20/10/2018,
27/10/2018,
03/11/2018,
10/11/2018,
17/11/2018,
24/11/2018,
01/12/2018,
08/12/2018,
15/12/2018,
22/12/2018,
25/08/2018,
01/09/2018,
08/09/2018,
15/09/2018,
22/09/2018,
29/09/2018,
06/10/2018,
13/10/2018,
20/10/2018,
27/10/2018,
03/11/2018,
10/11/2018,
17/11/2018,
24/11/2018,
01/12/2018,
08/12/2018,
15/12/2018,
22/12/2018,
25/08/2018,
01/09/2018,
08/09/2018,
15/09/2018,
22/09/2018,
29/09/2018,
06/10/2018,
13/10/2018,
20/10/2018,
27/10/2018,
03/11/2018,
10/11/2018,
17/11/2018,
24/11/2018,
01/12/2018,
08/12/2018,
15/12/2018,
22/12/2018,
25/08/2018,
01/09/2018,
08/09/2018,
15/09/2018,
22/09/2018,
29/09/2018,
06/10/2018,
13/10/2018,
20/10/2018,
27/10/2018,
03/11/2018,
10/11/2018,
17/11/2018,
24/11/2018,
01/12/2018,
08/12/2018,
15/12/2018,
22/12/2018,
