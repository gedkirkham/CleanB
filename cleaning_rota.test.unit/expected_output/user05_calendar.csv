Date,Kitchen,Bedroom_1,Bedroom_2,Bedroom_3,Bedroom_4,Bedroom_5,Lounge,Garage,Master Bedroom,
25/08/2018,Ged,Razvan,Tom,,,,,,,
01/09/2018,Razvan,,,Ged,Razvan,Razvan,Razvan,Razvan,Razvan,
08/09/2018,Tom,Tom,,Razvan,,,,,Tom,
15/09/2018,Ged,,,Tom,Razvan,,,,Ged,
22/09/2018,Razvan,Ged,Ged,Ged,,,,,Razvan,
29/09/2018,Tom,,,Razvan,Tom,Tom,Tom,Tom,Tom,
06/10/2018,Ged,Razvan,,Tom,,,,,Ged,
13/10/2018,Razvan,,,Ged,Ged,,,,Razvan,
20/10/2018,Tom,Tom,Razvan,Razvan,,,,,Tom,
27/10/2018,Ged,,,Tom,Razvan,Ged,Ged,Ged,Ged,
03/11/2018,Razvan,Ged,,Ged,,,,,Razvan,
10/11/2018,Tom,,,Razvan,Tom,,,,Tom,
17/11/2018,Ged,Razvan,Tom,Tom,,,,,Ged,
24/11/2018,Razvan,,,Ged,Ged,Razvan,Razvan,Razvan,Razvan,
01/12/2018,Tom,Tom,,Razvan,,,,,Tom,
08/12/2018,Ged,,,Tom,Razvan,,,,Ged,
15/12/2018,Razvan,Ged,Ged,Ged,,,,,Razvan,
22/12/2018,Tom,,,Razvan,Tom,Tom,Tom,Tom,Tom,
